<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuALL</class>
 <widget class="QWidget" name="MenuALL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Visualization</string>
  </property>
  <widget class="QFontComboBox" name="Font">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TextSize">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>34</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>36</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>34</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>36</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="AlignRight">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#AlignRight{
border-image:url(:/Icon/right.PNG);
border:none;
}
#AlignRight:hover{
    background-color: rgb(170, 255, 255);
    color:rgb(170, 255, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AlignCenter">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#AlignCenter{
border-image:url(:/Icon/center.PNG);
border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AlignLeft">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#AlignLeft{
border-image:url(:/Icon/left.PNG);
border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="Search">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>A1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AlignTop">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#AlignTop{
border-image:url(:/Icon/AlignTop.PNG);
border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AlignMedle">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#AlignMedle{
border-image:url(:/Icon/AlignMidle.PNG);
border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AlignBottom">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#AlignBottom{
border-image:url(:/Icon/AlignBottom.PNG);
border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Bold">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#Bold{
border:none;}</string>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Italic">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#Italic{
border:none;
}</string>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Underline">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#Underline{
border:none;}</string>
      </property>
      <property name="text">
       <string>U</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="Background">
      <property name="text">
       <string>A</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::MenuButtonPopup</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="TextColor">
      <property name="text">
       <string>a</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::MenuButtonPopup</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Function" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="graph">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="Barchart">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="LineChart">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="PieChart">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="ScatterChart">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
  </widget>
  <action name="actionHello">
   <property name="text">
    <string>Hello</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Function</class>
   <extends>QWidget</extends>
   <header>function.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
